### ISSUE TITLE
🐛 Bug Report: Backend Module Template should export default mod

### ISSUE THREAD

**Issue by tudi2d** [backstage/backstage#22178]
### 📜 Description

There is a new template for creating backend module templates for the new backend system. Currently there are some smaller issues with it one being that the module is not exported as default.

---

A possible fix to this could be to change https://github.com/backstage/backstage/blob/master/packages/cli/templates/default-backend-module/src/index.ts.hbs#L8 to: 

```ts
export { {{moduleVar}} as default } from './module';
```

### 👍 Expected behavior

Reading the documentation on building a module (https://backstage.io/docs/backend-system/building-plugins-and-modules/index#modules) and following the existing modules the expectation is that the exported as default. Each module package should only contain a single module, but this module may extend multiple extension points, so there should not be a need for a named export.

### 👎 Actual Behavior with Screenshots

Currently the module is not exporetd as a `default`: https://github.com/backstage/backstage/blob/master/packages/cli/templates/default-backend-module/src/index.ts.hbs 

```ts
export { {{moduleVar}} } from './module';
```

### 👟 Reproduction steps

1. Run `yarn new` & select `backend-module`
2. A new backend module is scaffolded
3. The module won't be exported as default from the package, which is inconsistend with the expectations

### 📃 Provide the context for the Bug.

cc: @Rugvip, @freben 

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by gauriimaheshwarii** [comment]
Hi @tudi2d, can I take this up?

**Comment by tudi2d** [comment]
Yes, please - go ahead & reach out if you need further guidance! Thanks!

**Comment by gauriimaheshwarii** [comment]
hey, I made a PR for the issue.
#22264

**Comment by tudi2d** [comment]
Closed by #22264
